### PR TITLE
Add mapbender commands as an example to use mapbender commands and a …

### PR DESCRIPTION
**Pull request by AxxL**
…first draft for a developers documentation directory

With this PR I want to introduce a doc directory to the mapbender-starter where we can add some developer-documentation that comes out of our CONTRIBUTING.MD. This documentation is a contrast to the user-documentation in doc.mapbender.org and should be written by the devs as fast and nice as possible, in the suitable MD format (which is not suitable for the user-documentation) and in a directory that can sum other parts of documentation as needed (Branching, Project-Creating, Building, Integrating in Gitlab-CI, etc.)

### CODE REVIEW

File: doc/mapbender_terminal_commands.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
+# Mapbender terminal commands
+Documentation of the Mapbender terminal commands needed for quick installation and updating within Composer.json use.
+
+Since Mapbender has implemented the composer tool, you can do all Composer editing from the console as well. The following commands are a selection that can be done via the console.
+
+If you look at all the commands and need the description, you can go to the page [Doku](https://getcomposer.org/doc/03-cli.md).
+
+# Creating a default composer.json
+The command `init-example` is used to create a composer.json, which is extracted from the bootstrap [Configuration](https://github.com/mapbender/mapbender-starter/blob/release/3.0.6/bootstrap). It is made an adjustment and overwrites the old composer.json a change and re-created. The composer.json is then created in the working directory.
```
Comment (DeEgge) [inline doc/mapbender_terminal_commands.md]: ... Adjustments are made, and the old composer.json is being overwritten and re-created in the working directory.

File: doc/mapbender_terminal_commands.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+ [OK] All assets were successfully installed.                                                                           
+
+> ComposerBootstrap::prepareBinaries
+> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installRequirementsFile
+
+```
+
+#### Update without dependencies
+
+If only the dev-require packages are to be updated, then you can optionally specify the parameter `--no-dev`. It disables the installation of require-dev packages.
```
Comment (DeEgge) [inline doc/mapbender_terminal_commands.md]: Doesn't --no-dev skip packages listed under 'require-dev'? If so, it should be "If only the non-dev/production dependencies are to be updated..."

File: doc/mapbender_terminal_commands.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
+# Mapbender terminal commands
+Documentation of the Mapbender terminal commands needed for quick installation and updating within Composer.json use.
+
+Since Mapbender has implemented the composer tool, you can do all Composer editing from the console as well. The following commands are a selection that can be done via the console.
+
+If you look at all the commands and need the description, you can go to the page [Doku](https://getcomposer.org/doc/03-cli.md).
```
[the remark this answers is not in the document]
Comment (AxxL) [reply doc/mapbender_terminal_commands.md]: OK

File: doc/mapbender_terminal_commands.md (hunk 1 above)
Comment (AxxL) [reply doc/mapbender_terminal_commands.md]: OK

File: doc/mapbender_terminal_commands.md (hunk 2 above)
Comment (AxxL) [reply doc/mapbender_terminal_commands.md]: Thanks. Right.

**Comment by AxxL** [comment]
Put in the changes of @DeEgge

**Pull request merged**